nengo_loihi/conv.py

```diff
@@ -242,4 +243,10 @@ def conv2d_loihi_weights(transform):
         inds = indices[axon_to_weight_map[ij]]
         assert (offsets[ij] + inds < n_compartments).all()
 
+    if len(np.unique([index_offset for _, _, index_offset in weights_map])) > 1:
+        warnings.warn(
+            "Compartment offsets exceed 256, and extra weights have been added to "
+            "facilitate this. To reduce the number of weights, keep the offsets < 256."
```

Comment:
I think we probably want to avoid taking on that extra work at this point. But then the question is, is it better to give the warning (and maybe it's wrong), or not give a warning (and then if the splitter doesn't fix it they might be using extra weights without realizing it).  I think I lean towards the latter, just because I picture it being a pretty frustrating experience getting this warning, trying to figure out what it means and how to fix it, and then realizing in the end that the warning wasn't actually relevant.

Reply:
I agree. I would rather have no warning than an inaccurate warning. If we want, we could have a "Troubleshooting" or some similar section, with an entry about "if you have too many weights, check this ..." and list the things I listed above.